openvpn/installpkg/recipes/Rakefile

```diff
@@ -0,0 +1,27 @@
+require 'rake'
```

Comment:
What is this file doing exactly?

Reply:
It's to use with serverspec-init for a manual test. With "kitchen verify", I see this is unnecessary and will be deleted.